lib/src/linden/xcolors.dart

```diff
@@ -388,6 +394,9 @@ class NewXColors extends XColors {
 
   @override
   Color get personalAreaSettings => _theme(bright: _purple80);
+
+  @override
+  Color get collectionsScreenCard => _theme(bright: _lightPurple);
```

Comment:
The change is fine, but I need to raise a general issue with the linden library. 
`collectionsScreenCard` is a presentation specific value, where as the design repo strives to be an independent lib that can be used for different projects. 
This is difficult to solve but we might think about two priciples:

- Move UI code to design when we also declare colors / themes specific for them (this means we need to make collections so generic that they can be used in the linden lib)
- Add a way to create Linden extensions in the target lib (I guess extending the Linden lib could already help, but not sure), so we can keep the Specific colors / themes in the target project and don't polute the Linden lib here.

@pro100svitlo  and @carminetoriello  Wdyt?

Reply:
Yeah true, we should aim to keep the library totally independent from the projects that use it!!
Regarding the first point, maybe we could implement into the library a generic `CardWidget` that comes already with the background color (like CardWidget.purple, CardWidget.orange etc), border radius etc that can be extended and used in different projects :)

Reply:
@joecks agreed. At some point we lost perspective 😢 
So far I do not see any value from this lib at all (more problems then benefits for now). Maybe if in the future we will start another project - it might be beneficial...

Yes, I was also thinking about creating a `generic-generic` Linden object, and later our `R` can extend it, and work with more extended `Colors`, `Dimens`, etc.
We will soon update this library with new fonts and other _tokens_ (slang from black team), so it might be a good point to start this refactoring.

Reply:
Just for reference: since the change is not specific only for this PR, it will be addressed later :)